src/renderer/pages/Project.tsx

```diff
@@ -25,17 +25,20 @@ const ProjectPage = () => {
 
   const handleOpenProject = async () => {
     try {
-      const project = await window.electron.openProject();
-      dispatch(projectOpened(project));
+      const { project, filePath } = await window.electron.openProject();
+      dispatch(projectOpened(project, filePath));
     } catch (err) {
       console.error(err);
     }
   };
 
+  const handleSaveProject = async () => {
```

Comment:
Do we still need these buttons in the project screen? Isn't the logic now integrated into the file menu?

Also, I believe they have been removed in Team Shrek's feature/edit-transcription-interface PR.

Reply:
Nah we dont I'll remove

Reply:
Done